src/components/Dashboard/Form/Applicants/Applicant.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import FieldWrapper from "../FieldWrapper/FieldWrapper";
+import styles from "./Applicant.scss";
+
+class Applicant extends Component {
+  state = {
+    isVisible: false
+  };
+  handleOnButton = () => {
+    if (this.state.isVisible === false) {
+      this.setState({ isVisible: true });
+    } else {
+      this.setState({ isVisible: false });
+    }
+  };
+  render() {
+    return (
+      <div className={styles.Applicant}>
+        <button
+          onClick={this.handleOnButton}
+          type="button"
+          className={styles.developedDiv}
+        >
```

Comment:
Maybe a better solution is using existing component [Button](https://github.com/CodeForPoznan/alinka-electron/blob/master/src/components/Button/Button.js)

Reply:
yes, but I need different styles, and I don't know is possible overwrite styles.Button, and don't know how. Any idea ?

Reply:
Should we have a button with different styles than others in the application?  If so, maybe change in Button component  - reading styles from props and if there are not given, using default styles

Reply:
I will consult with @agatakozinska  how many different buttons we will need and if there will be more of them I will add ticket for that and change Button component